src/installation/guides/raspberry-pi.md

```diff
@@ -0,0 +1,201 @@
+# Raspberry Pi
+
+As of 2017-11-30, Void Linux provides prebuilt Raspberry Pi images and rootfs
+tarballs. Compatible images and tarballs exist for RPi 1, RPi 2 and RPi 3, in
+both glibc and musl variants.
```

Comment:
I don't believe this infomation needs to be included here. What do you think?

Reply:
Most of this info is listed in the download page.

Reply:
Removed.